en_US/changes/changes-ee-5.3.md

```diff
@@ -1,5 +1,59 @@
 # Releases
 
+## e5.3.1
+
+### Enhancements
+
+- [#11637](https://github.com/emqx/emqx/pull/11637) Added extra diagnostic checks to help debug issues when mnesia is stuck waiting for tables. Library Updates: `ekka` has been upgraded to version 0.15.15, and `mria` to version 0.6.4.
+- [#11581](https://github.com/emqx/emqx/pull/11581) Preview Feature: Support for Version 2 Bridge Design with introducing a new "connector" concept.
+  - In the original Bridge v1 design, each connector was exclusively associated with a single bridge. While this design prioritized error isolation and performance, it posed challenges for users who needed to set up multiple bridges to the same service.  For example, setting up 10 bridges to a single Kafka cluster required the same configuration to be repeated for each bridge.
+  - The revamped Bridge v2 design provides more flexibility and better scalability. Users have the option to either share a connector among multiple bridges or assign it exclusively to one bridge, as in the v1 design. For most of the data bridges, while sharing a connector among too many bridges may lead to performance degradation, it helps avoid overloading external systems with numerous connections when managing a high number of bridges. For certain data bridges, dedicated connections are always used, even when the connector is shared. The Bridge v2 design has been introduced for the following data bridges:
```

Comment:
```suggestion
  - The new design introduces a significant improvement in flexibility and scalability. Now, users can configure a single "connector" to be used by multiple "actions", or continue with a one-to-one relationship as per the old design. This approach is particularly beneficial when dealing with a large number of actions to avoid overwhelming external systems with too many connections. A single connector can now serve multiple actions, or be dedicated to just one, as was the case in the Version 1 design. This can prevent the overloading of external services with connections, especially when managing numerous bridges. Certain data bridges will maintain dedicated connections for specific actions, even when utilizing shared connectors. The Version 2 Bridge Design has been introduced for the following data bridges:

    Kafka Producer
    Azure Event Hub Producer
```